recipes/subversion/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "subversion" %}
```

Comment:
In `defaults` this is named [`svn`]( https://anaconda.org/anaconda/svn ).

Reply:
In defaults, `svn` in the Python subversion wrapper, which is why I called 
 this `subversion` https://anaconda.org/anaconda/svn